plugin/fzf.vim

```diff
@@ -96,7 +96,13 @@ function! fzf#shellescape(arg, ...)
   if shell =~# 'cmd.exe$'
     return s:shellesc_cmd(a:arg)
   endif
+try
+  let oldshell = &shell
+  let &shell = shell
   return s:fzf_call('shellescape', a:arg)
+finally
+  let &shell = oldshell
+endtry
```

Comment:
* Unindented `try` block in the middle of a function doesn't read nicely, so indent the code one level.
* Multiple assignment to swap values

```suggestion
  try
    let [shell, &shell] = [&shell, shell]
    return s:fzf_call('shellescape', a:arg)
  finally
    let [shell, &shell] = [&shell, shell]
  endtry
```